docs/notebooks/4.1 Setting up and Using PV Sites.ipynb

```diff
@@ -0,0 +1,521 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "id": "593a3550-a145-4a2b-9c3d-7638c5c42b51",
+   "metadata": {},
+   "source": [
+    "The SDK provides '[rooftop](https://solcast.com/rooftop-pv-power-model)' and '[advanced](https://solcast.com/advanced-pv-power-model)' PV power estimates.  The advanced estimates use a more sophisticated PV model with comprehensive and flexible site configuration options.  Advanced PV power estimates are available for the [`live`](https://solcast.github.io/solcast-api-python-sdk/live/) and [`forecast`](https://solcast.github.io/solcast-api-python-sdk/forecast/) modules.  They require that a PV Power Site has been set up in the Solcast API to specify the configuration.  Site configurations can be managed via the [`pv_power_sites`](https://solcast.github.io/solcast-api-python-sdk/pv_power_sites/) module.   Calls to functions in this module do not consume any of your API request quota.\n",
+    "\n",
+    "In this example, we will use the SDK to\n",
+    "\n",
+    "* obtain some forecast PV power estimates for an existing site\n",
+    "* create a new PV Power Site configuration\n",
+    "* obtain some live PV power estimates for the site\n",
+    "* modify the site configuration\n",
```

Comment:
Capitilise new lines.

Align these dots points closer to the headings used below, so they are easier to cross-reference for a reader.

Reply:
done